app/code/community/Vindi/Subscription/Model/PaymentMethod.php

```diff
@@ -100,7 +100,7 @@ protected function createPaymentProfile($customerId)
 			'card_cvv'             => $payment->getCcCid() ?: '000',
```

Comment:
O que significa `000` ?

Reply:
são só zeros mesmo.
Se não for informado nenhum CVV serão enviados zeros :shrimp:

Reply:
Existe alguma razão para você estar resolvendo os meus comentários, antes de eu as respostas?

Adicionando outro comentário: Este conjunto de `3 zeros`, representa algum tipo de informação?

Reply:
Vocês podem atribuir esse conjunto de zeros a uma variável ou constante, por exemplo, dummy_cvv or fake_cvv assim deixa mais claro do que se tratam os zeros

Reply:
Atualizado (CC @thiagopecanha)

Reply:
> Adicionando outro comentário: Este conjunto de 3 zeros, representa algum tipo de informação?

Não, na verdade é apenas para que o CVV não seja enviado como `null` e, para isso, enviamos os zeros